proposal/Eric860730/README.rst

```diff
@@ -0,0 +1,119 @@
+============================
+NSD Term Project
+============================
+
+Basic Information
+----------------------------
+| Github repository url:
+| https://github.com/Eric860730/A-simple-virus-spread-simulation
+
+Problem to Solve
+----------------------------
+| In recent years, COVID-19 has been raging around the world.
+| It has a great impact on everyone's lives.
+| According to this article[1], the spread of COVID-19 is increasing exponentially.
+| In order to prevent the spread of COVID-19, each country/region has implemented many policies (e.g. quarantine, restriction of in-store dining, lockdown...).
+| These policies have produced different results in controlling the spread of COVID-19.
+
+
+| I want to simulate the impact of different policies on the spread of pandemic viruses.
+ User can adjust parameters(e.g. virus infection rate, virus mortality rate...) and policy to observe the final result.
+ I will use animation to show the process of infection and print the distribution of the virus spread.
+| Therefore, we can observe the difference in the distribution of infection under different policies.
+
+Prospective Users
+----------------------------
+| Those who want to know if these policies help prevent the spread of COVID-19.
+| Users can set different conditions, such as the total number of people, virus infection rate, death rate, and different policies.
+| Then observe the distribution of the virus spread and the number of deaths.
+
+System Architecture
+----------------------------
+
+Input
+^^^^^^^^^^^^
+* total number of people
+* moving speed of people
+* virus infection rate
+* virus mortality rate
+* recovery time
+* choose policy (Free, Attempted quarantine, Moderate distancing, Extensive distancing)
+* healthcare capacity (The mortality rate is halved.)
+
+Output
+^^^^^^^^^^^^
+* Animation of the virus spreading process.
+* The distribution graph of virus spread.
+* The number of uninfected people.
+* The number of recovered people.
+* The number of death people.
+
+System Components
+^^^^^^^^^^^^^^^^^
+1. Parser: Python class. Parse user input parameters for simulation.
+2. Simulator: C++ class. Use input parameters to simulate the spread of the virus.
+3. Printer: Python class. Print simulation results with animation.
+
+System Workflow
+^^^^^^^^^^^^^^^^^
+.. image:: images/system_workflow.png
+
+Constraints
+^^^^^^^^^^^^^^^^^
+1. Users cannot add their own policies.
```

Comment:
I added a new API called “customizePolicy”. User can control their own policy through input the order parameter. 
I am not sure if this is custom logic? If not, what does custom logic mean and where should I start?

Reply:
No, a distinct API is not the customization I meant.  It may be a little bit difficult to quickly show you how the custom API look like, but I plan to use the second half of the course to do it.  A short answer is to turn your system into a library.

Reply:
I am resolving this discussion because this is something you need to spend more time on.

Reply:
Okay, I will implement my proposal first. If I have enough time and knowledge, I will try to implement custom logic. 
Thank you professor for your guidance.